mac

```diff
@@ -166,7 +166,14 @@ brew_install_or_upgrade 'hub'
 # shellcheck disable=SC2016
 append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
 
-brew_install_or_upgrade 'node'
+if ! command -v n > /dev/null; then
+  fancy_echo 'Installing n and latest Node.js and NPM...'
+  curl -L http://git.io/n-install | bash -s -- -y latest
+else
+  fancy_echo 'Updating n...'
+  n-update -y
+fi
+
```

Comment:
You mentioned that there is a "requirement [...] to add it's directory as N_PREFIX, and add $N_PREFIX/bin to your $PATH". Should the script also update `.zshrc` to add that to the PATH?